### ISSUE TITLE
Preprocessing on sentiment data

### ISSUE THREAD

**Issue by KamelGaanoun** [alger-ia/dziribert#3]
Hi, great work!

Didn't you performed same preprocessing steps on sentiment data as you did for pre training data?
I see in the evaluate_model.py that raw data is used.

Thanks

**Comment by amineabdaoui** [comment]
Thanks @KamelGaanoun 

As you mentioned we haven't applied the same pre-processing steps on the evaluation data. We wanted to have the same script for all the models and not to advantage our own model. 

At the same time, I believe that the pre-processing steps applied during the pre-training are mainly targeting the anonymization side while avoiding to change the original nature of the entries (e.g. the replacement of an email is still an email).

Sorry for my late reply !
